src/main/java/org/graylog/integrations/notifications/types/SlackEventNotificationConfig.java

```diff
@@ -104,6 +111,15 @@ public abstract class SlackEventNotificationConfig implements EventNotificationC
     @Nullable
     public abstract String iconEmoji();
 
+    @JsonProperty(FIELD_TIME_ZONE)
+    public abstract DateTimeZone timeZone();
```

Comment:
@kingzacko1 I'm getting an error if I remove the time zone from the config and then execute a test notification (slack and teams). 

I'm also noticing that if I select `GMT-12` in the config I get `-12` in the resulting notification (and vice versa). Not sure if this is just me misunderstanding how that's supposed to work and you didn't touch any of that conversion code here.

```suggestion
    @Nullable
    public abstract DateTimeZone timeZone();
```
![image](https://user-images.githubusercontent.com/103449971/223192121-cf7d9c7e-ecf2-4d34-b647-beb59a4e61fa.png)

Reply:
The idea was to require a time zone, but just default to UTC (as it has always been) if the field is never set. Let me remove the ability to clear the time zone. 

The `GMT-12` one may be a different issue, but I will check it out.

Reply:
Ok makes sense to me. Just a heads up in case you're not already tracking, the UI says `(optional)` for that field so that should probably go too.

Reply:
@ryan-carroll-graylog and I don't understand it either... but look at [this list](https://www.ibm.com/docs/en/cloudpakw3700/2.3.0.0?topic=SS6PD2_2.3.0/doc/psapsys_restapi/time_zone_list.htm) of time zones and see that `Etc/GMT-12` is GMT+12 and `Etc/GMT+12` is GMT-12.... ¯\\\_(ツ)_/¯

Reply:
Ah well there ya go. Figured it was something like that, with the alternative being that it been broken everywhere else we're doing similar and nobody noticed.